test/integration/package_int_test.go

```diff
@@ -278,9 +278,9 @@ func (suite *PackageIntegrationTestSuite) TestPackage_detached_operation() {
 	cp.ExpectExitCode(0)
 
 	suite.Run("install non-existing", func() {
-		cp := ts.Spawn("install", "json")
+		cp := ts.Spawn("install", "argpars")
```

Comment:
See https://activestatef.atlassian.net/browse/DX-2805?focusedCommentId=53790 for an explanation of why this changed.